tests/device_selection/test_predictor_ml.py

```diff
@@ -91,9 +91,9 @@ def test_save_training_data(predictor: ml.Predictor, source_path: Path, target_p
         assert path.exists()
 
 
-def test_train_random_forest_classifier_and_predict(predictor: ml.Predictor, source_path: Path) -> None:
+def test_train_random_forest_model_and_predict(predictor: ml.Predictor, source_path: Path) -> None:
```

Comment:
We should definitely add tests for the new figure of merit. Especially since the fact that the trained model is now a regressor instead of a classifier might cause unexpected side effects. Therefore, the training and usage of a small model should be covered - ideally for the RL part as well.